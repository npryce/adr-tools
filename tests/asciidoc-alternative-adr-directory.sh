adr init -t adoc alternative-dir
adr new Example ADR

ls .
ls alternative-dir

adr list
