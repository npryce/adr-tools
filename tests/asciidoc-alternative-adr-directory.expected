adr init -t adoc alternative-dir
alternative-dir/0001-record-architecture-decisions.adoc
adr new Example ADR
alternative-dir/0002-example-adr.adoc

ls .
alternative-dir
ls alternative-dir
0001-record-architecture-decisions.adoc
0002-example-adr.adoc

adr list
alternative-dir/0001-record-architecture-decisions.adoc
alternative-dir/0002-example-adr.adoc
